quint/src/graphics.ts

```diff
@@ -187,7 +187,7 @@ export function prettyQuintType(type: QuintType): Doc {
       return group([text('{ '), prettyRow(type.fields), text('}')])
     }
     case 'sum': {
-      return group([text('{ '), prettySumRow(type.fields), text('}')])
+      return prettySumRow(type.fields)
```

Comment:
This is cleanup from the previous PR adding the sum types. The were wrapped in an extra `{...}`